scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
```

Comment:
Maybe add `After=posgresql` as well. I *think* as long as you leave of the requires, it won't actually cause postgres to be started, but if postgres is set to start, it will make things come up in the right order. Probably deserves a comment explaining why it's there, and we'll want to add any additional supported DBMSes in the future.